libs/barista-components/filter-field/src/filter-field.ts

```diff
@@ -579,6 +585,20 @@ export class DtFilterField<T = any>
       .subscribe(() => {
         this._currentTags.next(this._tags.toArray());
       });
+
+    this._autocomplete.opened
+      .pipe(startWith(null), takeUntil(this._destroy$))
+      .subscribe(() => {
+        this.interactionState = true;
+        this.interactionStateChange.emit(true);
+      });
+
+    this._autocomplete.closed
+      .pipe(startWith(null), takeUntil(this._destroy$))
+      .subscribe(() => {
+        this.interactionState = false;
+        this.interactionStateChange.emit(false);
+      });
```

Comment:
I don't think this covers the whole range of the interaction. The autocomplete can be closed in this case if the current filter is a range (or multi-select later on when #1836 is done). 

Additionally, a autocomplete does not have to be open while interacting (free text filter).